crates/ty_python_semantic/src/types.rs

```diff
@@ -6860,6 +6875,7 @@ impl<'db> FunctionType<'db> {
         Type::Callable(CallableType::from_overloads(
             db,
             self.signature(db).overloads.iter().cloned(),
+            false,
```

Comment:
It seemed somewhat tempting to return `true` here :smile:. And maybe that would be more correct in some sense? At the moment, I think it does not make any difference, since `FunctionType::into_callable_type` is only used in type-property functions that do not seem to invoke the descriptor protocol.

Reply:
If it doesn't change any of the current visible behavior I think I would prefer `true` here, if we're using that field to indicate whether the callable binds self when participating in the descriptor protocol.